src/api/lib/data.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs'
+import { DB, ServiceKey } from '../types/admin'
+
+const data: DB = {
+  serviceKeys: []
+}
+
+const getData = (serviceKeys: string) => {
+  if (serviceKeys) {
+    fs.readFile('./data.db', (err, file) => {
```

Comment:
does not make sense to `readFile` every time `getData` is called. Please review JS4 problem 8. This is exactly that exercise.

Reply:
But in case you don't remember, here's how you should implement `getData`:
```
const getData = (table: string) => {
  return data[table]
}
```

Move your `fs.readFile` outside of any function calls, so whenever the file loads you automatically populate data into memory.